public/api/conf/1.0/docs/testapi.md

```diff
@@ -1 +1,15 @@
-You can use the sandbox environment to [test this API.](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing)
\ No newline at end of file
+You can use the sandbox environment to test this API. Please note that due to the size of the files, we have been unable to integrate these to our sandbox environment, however this environment can be used to test your integration with the API.(https://developer.service.hmrc.gov.uk/api-documentation/docs/testing)
+
+On the 1st of January each year, the download pattern you need to follow is as below:
+
+Download annual file.
+Download daily deltas to present date.
+Continue with daily deltas from then on.
+
+The monthly roll-up files will be required if you sign up in the months after the annual file is first available:
+
+
```

Comment:
Too many empty lines?

Reply:
Ok. I will remove it

Reply:
Not sure how will look if remove all spaces. I will merge it and see how it looks like then will incorporate the changes